corehq/apps/users/views/mobile/users.py

```diff
@@ -756,11 +763,61 @@ def create_mobile_worker(self, in_data):
             phone_number = self.new_mobile_worker_form.cleaned_data['phone_number']
             couch_user.set_default_phone_number(phone_number)
             send_account_confirmation_sms_if_necessary(couch_user)
+
+        data = self.get_plan_and_user_count_by_domain(self.domain)
+        self.check_and_send_limit_email(self.domain, data['plan_limit'], data['user_count'])
         return {
             'success': True,
             'user_id': couch_user.userID,
         }
 
+    @staticmethod
+    def get_plan_and_user_count_by_domain(domain):
```

Comment:
I don't think this belongs in a view module. Re-usable computation seems more like a model-level concern, and I think having our modules import other view code tends to lead to circular dependency issues.

Reply:
Moved to be a classmethod of the Subscription class